### ISSUE TITLE
Can't install Install/Update (Full Rom) Firmware on my Dell Chromebook 11 3189 

### ISSUE THREAD

**Issue by amdiebs** [MrChromebox/firmware#699]
Hello, Mr.
I would like to reinstall my laptop's operating system from ChromeOS to Windows 10. Unfortunately, an error '(see screenshot) appeared during the installation of the Full ROM firmware after all the preliminary steps had been performed (remove screws, connect in developer mode) before, and it was not possible to continue the procedure.
I need help with this.
![Screenshot 2024-11-12 11 50 24](https://github.com/user-attachments/assets/07fdff64-99c5-4706-87ac-096ae333da04)

PS: file flashrom.log is joint for more details. Thanks
[flashrom.log](https://github.com/user-attachments/files/17716206/flashrom.log)

**Comment by MrChromebox** [comment]
it's not clear why the flash failed, I would restore the stock firmware manually, reboot, and then try flashing again.

restore manually:
`sudo flashrom -p internal -w /tmp/bios.bin --ifd -i bios -N`

**Comment by amdiebs** [comment]
I tried the command, with success (1st screenshot)![Screenshot 2024-11-12 16 52 21](https://github.com/user-attachments/assets/d57e9f9d-1f49-4565-b5d2-c0ff0e917cc9)
But if I try flashing again, the same errors as before reappear.
see screenshots below
![Screenshot 2024-11-12 16 45 32](https://github.com/user-attachments/assets/0cf2054a-a49c-4660-946b-b8d83e36a53f)
![Screenshot 2024-11-12 16 46 56](https://github.com/user-attachments/assets/7e093725-315a-45a5-ad87-0bebb691d3f4)

**Comment by MrChromebox** [comment]
you rebooted after restoring the stock firmware and re-attempting the flash?

**Comment by amdiebs** [comment]
I didn't mention this but I did reboot after the restore before trying flashing again.
I did it again to reassure you, but same error. 
just feel like crying...

**Comment by MrChromebox** [comment]
> just feel like crying...

because your Chromebook is having an issue flashing the firmware?

**Comment by amdiebs** [comment]
Yes, because of that, it was supposed to be a formality. But ultimately time-consuming.

**Comment by MrChromebox** [comment]
please provide the output of `sudo flashrom -p internal --wp-status`

**Comment by amdiebs** [comment]
![Screenshot 2024-11-13 16 17 59](https://github.com/user-attachments/assets/c114e22b-59b6-4d74-b61b-d70fb353e694)

**Comment by amdiebs** [comment]
you saw finally what's wrong?

**Comment by MrChromebox** [comment]
no, otherwise I would have replied. I'm going to go back to stock on one of my Braswell devices and try to reproduce this issue

**Comment by foxboy2005** [comment]
I am also having the same issue. I am hoping for the solution ASAP. Thank you Mrchromebox!

**Comment by MrChromebox** [comment]
I just reset my EDGAR board back to stock and flashed via the script and no issue at all. Checked the flashrom log and no issues there. Comparing the log provided by the OP, it seems they are using an older version of ChromeOS/older version of flashrom. Also, the boards use a different flash chip
